pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -148,6 +148,15 @@
         doc = "Path for the file used to determine the rotation status for the proxy instance"
             + " when responding to service discovery health checks"
     )
+    private Integer proxyLogLevel = 0;
+
+    @FieldContext(
```

Comment:
I think the `@FieldContext` was added to the wrong place. it should be for `proxyLogLevel`, and the one for `proxyLogLevel` should be for `statusFilePath`.